src/WebJobs.Script/Utility.cs

```diff
@@ -11,6 +11,7 @@
 using System.Linq;
 using System.Reflection;
 using System.Text;
+using System.Text.RegularExpressions;
```

Comment:
Remove unused imports

Reply:
Any specific ones you had in mind? The `System.*` ones here seem OK to me